ui/src/adapters/api/credentials.ts

```diff
@@ -518,6 +518,46 @@ export async function createAuthQRCode({
   }
 }
 
+type ResultOKUnknown = {
+  data?: unknown;
+  status: 200;
+};
+
+const resultGetQRParser = getStrictParser<ResultOKUnknown>()(
```

Comment:
`resultOKUnknownParser`. What we parse (`<ResultOKUnknown>`) plus the suffix `parser`